troupon/static/scss/partials/_typography.scss

```diff
@@ -0,0 +1,47 @@
+
+// ==========================================================================
+//  Typography
+// ==========================================================================
+
+body {
+  font-family: font-family(primary);
+}
+
+@each $tag, $font-weight in (h1: light, h2: regular, h3: bold, h4: bold, body: regular) {
+  #{$tag} {
+    color: color(#{$tag});
+    font: {
+      size: font-size(#{$tag});
+      weight: font-weight(#{$font-weight});
+    }
+  }
+}
+
+h1 {
+  letter-spacing: -1px;
+}
+
+.mini-text {
+  font-size: font-size(mini);
+}
+
+
+.mini-label {
+  font: {
+    size: font-size(label);
+    weight: font-weight(#000000);

```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.